### ISSUE TITLE
Exit key setting to a list of keys doesn't work

### ISSUE THREAD

**Issue by Twix53791** [liuchengxu/vim-which-key#244]
**Environment:**
 - OS: Arch linux

 - (Neo)Vim version: 0.9.1
    0b901c9

 - mini init.vim
```vim
call plug#begin('~/.config/nvim/plugins')

Plug 'liuchengxu/vim-which-key'

call plug#end()

let g:which_key_map =  {}
let g:mapleader = ","
nnoremap <silent> <leader> :<C-u>WhichKey ','<CR>
call which_key#register(',', "g:which_key_map")

let g:which_key_exit = ["\<Esc>", "\<S-CR>", "\<A-CR>"]
```

**Describe the bug**
When I put the keys into a list, none of each works, to exit the menu I have to select an entry. Curiosly, each key set separatly like `let g:which_key_exit = "\<S-CR>"`, outside of a list, works.

**To Reproduce**

1. Create the minimal vimrc `min.vim`:
```vim
call plug#begin('~/.config/nvim/plugins')

Plug 'liuchengxu/vim-which-key'

call plug#end()

let g:which_key_map =  {}
let g:mapleader = ","
nnoremap <silent> <leader> :<C-u>WhichKey ','<CR>
call which_key#register(',', "g:which_key_map")

let g:which_key_exit = ["\<Esc>", "\<S-CR>", "\<A-CR>"]
```

**Expected behavior**
I found on github many personal vim configs setting which_key_exit as a list of keys, so I am surprised it doesn't functionate on my config...

**Comment by liuchengxu** [comment]
There is indeed a regression here, which should be fixed by #246. However, keys like `"\<S-CR>"` are not guaranteed to work in vim, you have to use the distinguishable keys, please refer to https://stackoverflow.com/questions/5388562/cant-map-s-cr-in-vim for more info.

**Comment by Twix53791** [comment]
Thank you! the fix works for me! And in my config `"\<S-CR>"` works perfectly so I am fulfilled.
